src/components/Clusters/views/ClusterOverview/ClusterDetails.js

```diff
@@ -36,6 +38,30 @@ export default function ClusterDetails({ currentCluster }) {
   const { modules, error, loading: loadingModules } = useKymaModulesQuery();
   const { clusterUrl } = useUrl();
   const navigate = useNavigate();
+  const {
+    data: statuses,
+    loading: loadingStatuses,
+    error: statusesError,
+  } = useModulesStatuses(modules);
+
+  const moduleCounts = useMemo(() => {
```

Comment:
maybe name it `moduleStatusCounts` or `countModuleStatuses`, will be a little bit clearer